app/code/Codifi/CustomerRequest/etc/email_templates.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Email:etc/email_templates.xsd">
+    <template id="email_request_template" label="This is email demowed0q!!!!!" module="Codifi_CustomerRequest" file="email_request_template.html" type="html" area="frontend"/>
```

Comment:
1. Check the label
2. It's better to name template 'customer_request_template' as it's already template of email